frontend/src/services/communication/private/Auth.js

```diff
@@ -1,21 +1,56 @@
 import { useState } from 'react';
 
 class Auth {
-   constructor() {
+   constructor( backend ) {
```

Comment:
You have a circular dependency here as `Backend` requires `Auth` and `Auth` requires `Backend`.